src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1886,56 +1885,39 @@ TEST(SpatialPoolerTest, testSerialization2) {
 
   const UInt inputSize = 200;
   const UInt numColumns = 200;
-  const UInt w = 10;
```

Comment:
This test look broken.  It no longer sets the input before running the SP...
Maybe instead of shuffle you meant randomize?

Reply:
Thank you for spotting the mistakes in the tests! 
I'll go on and merge this with the former approval, assuming I just fixed the test.